package.json

```diff
@@ -38,7 +38,9 @@
   },
   "devDependencies": {
     "@apollo/client": "3.7.5",
+    "@apollo/gateway": "https://pkg.csb.dev/apollographql/federation/commit/166fdd24/@apollo/gateway",
     "@apollo/server-plugin-landing-page-graphql-playground": "4.0.0",
+    "@apollo/subgraph": "https://pkg.csb.dev/apollographql/federation/commit/166fdd24/@apollo/subgraph",
```

Comment:
Why is this needed?

Reply:
Probably could've always been just the current version of subgraph but this will be latest once the patch release is published